app/views/partials/_performance_metrics.html.erb

```diff
@@ -0,0 +1,12 @@
+<h2 class="govuk-heading-m"><%= t("performance_data.subtitle") %></h2>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-half govuk-grid-column-one-half-from-desktop">
+    <%= render MetricsSummaryComponent::View.new(@performance_data) %>
+  </div>
+</div>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <p>last updated: <%= @last_updated %></p>
+    <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
+  </div>
+</div>
```

Comment:
I think with the CSS change above this can be simplified by making the whole partial full width:
```suggestion
<div class="govuk-grid-row">
  <div class="govuk-grid-column-full">
    <h2 class="govuk-heading-m"><%= t("performance_data.subtitle") %></h2>
    <%= render MetricsSummaryComponent::View.new(@performance_data) %>
    <p>last updated: <%= @last_updated %></p>
    <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
  </div>
</div>
```

Visually I think this should do the job and be resilient to being placed in different layouts